beginner_source/transformer_tutorial.py

```diff
@@ -1,47 +1,45 @@
 """
-Sequence-to-Sequence Modeling with nn.Transformer and TorchText
+nn.Transformer 와 TorchText 로 시퀀스-투-시퀀스(Sequence-to-Sequence) 모델링
 ===============================================================
```

Comment:
사소하지만 reST(reStructuredText)에서는 제목보다 길게 라인을 넣어주도록 되어 있습니다.
이에, 아래와 같이 제안드립니다.
```suggestion
=========================================================================
```